tests/test_builders/test_build_html_image.py

```diff
@@ -72,6 +72,7 @@ def test_copy_images(app, status, warning):
     images_dir = Path(app.outdir) / '_images'
     images = {image.name for image in images_dir.rglob('*')}
     assert images == {
+        # 'ba30773957c3fe046897111afd65a80b81cad089.png',  # html: image from data:image/png URI in source
```

Comment:
Commented out?

Reply:
Yep; it's intended as a hint that the same image exists as in the other `test_copy_images` unit tests, but that it isn't written to file because HTML (and similarly ePub) can natively include `img src="data:..."` elements.

Probably not required, but if some future change affects the logic, it's possible that all of the relevant test cases would need to be updated, and in that case the cross-reference provided by the comment _might_ be useful.